internal/config/db_test.go

```diff
@@ -0,0 +1,178 @@
+//
+// Copyright 2023 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config_test
+
+import (
+	"context"
+	"database/sql/driver"
+	"errors"
+	"net/url"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stacklok/minder/internal/config"
+)
+
+// TODO: add this with a fake DB connection driver
+type fakeDriver struct {
+	conns string
+}
+
+var _ (driver.Driver) = (*fakeDriver)(nil)
+var _ (driver.DriverContext) = (*fakeDriver)(nil)
+var _ (driver.Connector) = (*fakeDriver)(nil)
+var _ (driver.Conn) = (*fakeDriver)(nil)
+
+func (f fakeDriver) Open(name string) (driver.Conn, error) {
+	return nil, errors.ErrUnsupported
+}
+func (f fakeDriver) OpenConnector(name string) (driver.Connector, error) {
+	return fakeDriver{name}, errors.ErrUnsupported
+}
+func (f fakeDriver) Connect(ctx context.Context) (driver.Conn, error) {
+	return &fakeDriver{f.conns}, nil
+}
+func (f fakeDriver) Driver() driver.Driver {
+	return &f
+}
+
+func (f *fakeDriver) Prepare(query string) (driver.Stmt, error) {
+	return nil, errors.ErrUnsupported
+}
+
+func (f *fakeDriver) Close() error {
+	return nil
+}
+
+func (f *fakeDriver) Begin() (driver.Tx, error) {
+	return nil, errors.ErrUnsupported
+}
+
+func TestDatabaseConfig_GetDBConnection(t *testing.T) {
+	t.Parallel()
+	// sql.Register("postgres", &fakeDriver{})
```

Comment:
Should this be commented like this?

Reply:
This was from some earlier work, and isn't needed at alle